### DISCUSSION TITLE
fluentPage does not accept shinyWidgets::pickerInput()

### DISCUSSION THREAD

**Discussion by asadow** [Appsilon/shiny.react#71] [General]
I noticed that when I had switched from using imola::gridPage() to shiny.fluent::fluentPage(), that the app did not work. IIRC the card disappeared and it was due to using shinyWidgets::pickerInput(). 

Does fluentPage() only accept fluent inputs?

**Comment by kamilzyla** [comment]
Hello @asadow! The `shiny.fluent::fluentPage()` is just a helper which wraps the tags passed as arguments in `<body>`, attaches Fluent UI dependencies and suppresses Bootstrap which can collide with Fluent UI styles. Non-`shiny.fluent` components should work just fine inside `fluentPage()`.

You might try calling it as `fluentPage(suppressBootstrap = FALSE, ...)`. If that doesn't help, please share a minimal code example of the issue you are facing and we'll investigate :slightly_smiling_face:

**Reply by asadow** [reply]
Hi, @kamilzyla! 

Good to know about the `suppressBootstrap` argument. I don't believe it will help however, as I was using `Stack()`, and regular shiny inputs are causing an error. This is reprexed in #70.

**Reply by kamilzyla** [reply]
Right, the root cause of your problem was likely #70. See my reply there and let us know if you face any new problems.
